glidertest/tools.py

```diff
@@ -1117,16 +1112,19 @@ def ramsey_binavg(ds, var='VERT_CURR', zgrid=None, dz=None):
     ----
     I know this is a non-sensical name.  We should re-name, but is based on advice from Ramsey Harcourt.
     """
-
+    _necessary_variables_check(ds, [var, 'PRES'])
     press = ds.PRES.values
     ww = ds[var].values
 
     # Calculate depth from pressure using gsw
     if 'DEPTH_Z' in ds:
         depth = ds.DEPTH_Z.values
-    else:
+    elif:

```

Comment:
I think this should be `elif 'LATITUDE in list(ds.variables)` or similar? Can't just have a plain `elif:` it needs a test like regular `if`: